app/views/mentor_overview/_participant_overview.html.haml

```diff
@@ -11,9 +11,9 @@
       %span.title= "#{student.first_name} #{student.last_name}"
       %p
         - open_responses.each do |response|
-          - response.initialize_invitation_token! if response.invitation_token.nil?
+          - next if response.expired?
```

Comment:
Dit is overbodig toch? `open_responses` staat gedefinieerd in regel 3 als `open_responses = protocol_subscription.responses.opened.reject{|resp| resp.expired?}`